recipes/libnl/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "libnl" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://www.infradead.org/~tgr/libnl/files/libnl-{{ version }}.tar.gz
+  sha256: 4f80c21fe5bbcdde6e72b59b4f98306063a41421f909887c34e58d93e746d063
+
+build:
+  skip: True  # [not linux]
+  number: 0
+  run_exports:
+    # https://abi-laboratory.pro/?view=timeline&l=libnl
+    - {{ pin_subpackage('libnl', 'x.x.x') }}
+  script:
+    - ./configure --prefix=$PREFIX
+    - make -j$CPU_COUNT
+    - make install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libnl.so
+    - test -f $PREFIX/include/netlink/types.h
+
+about:
+  home: https://www.infradead.org/~tgr/libnl/
+  license: LGPL 2.1
```

From [here](https://opensource.org/licenses/alphabetical):
```suggestion
  license: LGPL-2.1
```